### PR TITLE
Fix z-order of wall overlay elements, renamed files for consistency

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by rdumusc** [comment]
I haven't got a clue why find_package(MPI) no longer finds mpich on Travis; brew install mpich succeeds, and on my mac it works... :-(

**Comment by rdumusc** [comment]
Travis builds again, hoorah!

**Comment by rdumusc** [comment]
ping?

File: tide/master/control/DisplayGroupController.cpp [hunk 1]
```diff
@@ -48,10 +48,9 @@
 
 namespace
 {
-bool _exceeds(const auto& size, const auto& other)
-{
+static auto _exceeds = [](const auto& size, const auto& other) {
     return size.width() > other.width() && size.height() > other.height();
-}
+};
```
Comment (tribal-tec) [inline tide/master/control/DisplayGroupController.cpp]: ?

Comment (rdumusc) [reply tide/master/control/DisplayGroupController.cpp]: The explanation is in the commit message for this change, auto parameters for non-lambda functions is not in C++14 standard so clang refuses to compile it (unlike gcc). The solution is either this (a static lambda) or declare a standard template function (with typename XYZ).

Comment (tribal-tec) [reply tide/master/control/DisplayGroupController.cpp]: true

**Pull request merged**